Python/Email_Scrape_From_Commits/README.md

```diff
@@ -0,0 +1,25 @@
+# Email Scraper for commit history
+
+A simple python script to crawl all email address, name and username from a specific repository on GitHub.
+
+## Setup instructions
+
+| Install:          |
+| ----------------- |
+| `bash install.sh` |
+
+## Run instructions
+
+| Run:                                    |
+| --------------------------------------- |
+| `Python email_scrape.py --help`         |
+| `python email_scrape.py <token> <repo>` |
+
+## Author
+
+<b>[Rahul Raikwar](https://github.com/rahulraikwar00)</i>
+
+## Disclaimers, if any
+
+- <i>This script requires a Github Personal Access Token (PAT) to verify the user</i>
```

Comment:
Can we use `.md` syntax wherever possible?
Markdown generally isn't a fan of inline HTML
```
** for bold and
* for italics
```

Reply:
done 👍